tests/pipelines/stable_diffusion/test_stable_diffusion.py

```diff
@@ -154,6 +172,15 @@ def get_dummy_components(self, time_cond_proj_dim=None):
         )
         text_encoder = CLIPTextModel(text_encoder_config)
         tokenizer = CLIPTokenizer.from_pretrained("hf-internal-testing/tiny-random-clip")
+        feature_extractor = CLIPImageProcessor()
```

Comment:
once this PR https://github.com/huggingface/diffusers/pull/6868 is merged, IMO we can maybe skip this step ( i.e. creating `feature_extractor` and `image_encoder`) and test with `ip_adapter_image_embedding` directly. 
I think it is ok to test with image embeddings directly because the process to convert `ip_adapter_image` to `ip_adapter_image_embedding` is the same across all pipelines, using the same `prepare_ip_adapter_image_embeds` method; and It will be a lot easier to add this IP adapter test to a pipeline without this step.

but ok to keep it here for now and we can discussion more:)

Reply:
yes that works and will be a simpler addition with just needing to inherit this mixin to test a pipeline